Shoko.Server/Import/Importer.cs

```diff
@@ -271,7 +271,12 @@ public static void RunImport_ScanFolder(int importFolderID, bool skipMyList = fa
                         if (fldr.IsDropSource == 1)
                             dictFilesExisting[fileName].RenameAndMoveAsRequired();
                     }
-                    if (fileName.Contains("$RECYCLE.BIN")) continue;
+
+                    if (ServerSettings.Instance.Import.Exclude.Any(s => fileName.Contains(s)))
```

Comment:
Are we able to change this to a [Regex]? It would then allow us to handle a 'Contains' or an exact match.

[Regex]: https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex?view=netstandard-2.1

Reply:
In the settings would it make sense for it to allow a list of Regex strings or only allow one Regex string? , I ask because in a single Regex im fairly sure you can define mulitple conditions so to speak.

Reply:
For easier user understanding, do multiple